### DISCUSSION TITLE
Secret Achievements? (Or maybe planned but unreleased?)

### DISCUSSION THREAD

**Discussion by norwd** [Schweinepriester/github-profile-achievements#469] [Q&A] [answered]
I've found an official GitHub [doc page] ([archive][archived doc page]) the seams to include screenshots of two achievements that I could not find any reference too. Neither in this repo nor anywhere online.

The first image, [achievements-on-profile.png], seams to show two achievements, one with a tentacle wearing an armband with a heart emoji on it, and another with wizard/witch hat with angle brackets on it.

[![achievements-on-profile.png]][achievements-on-profile.png]

The second image, [achievements-hidden.png], show a different view, with both achievements visible but partially covering the wizard hat. This second image also shows that the achievement with the tentacle and heart emojis is called "Heart On Your Sleeve" and that the achievement with the hat is called "Open S..." with the remainder cut off. (My guess is that it might be called "Open Source Wiz"?).

[![achievements-hidden.png]][achievements-hidden.png]

Note that the above image is actually cropped like that.

Does anybody know anything about either of these? Were they achievements that used to exist but were scrapped or are they just so rare the nobody has them in the wild yet?

[doc page]: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile
[achievements-on-profile.png]: https://docs.github.com/assets/cb-187399/images/help/profile/achievements-on-profile.png
[achievements-hidden.png]: https://docs.github.com/assets/cb-412132/images/help/profile/achievements-hidden.png

[archived doc page]: https://web.archive.org/web/20221107225358/https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile
[archived achievements-on-profile.png]: https://web.archive.org/web/20221107225358im_/https://docs.github.com/assets/cb-187399/images/help/profile/achievements-on-profile.png
[archived achievements-hidden.png]: https://web.archive.org/web/20221107225358im_/https://docs.github.com/assets/cb-412132/images/help/profile/achievements-hidden.png

**Comment by Schweinepriester** [comment, marked as answer]
Oha, very well spotted, thank you!

I had speculated that they are going to release more achievements as part of github universe (starting today!), so I‘m willing to bet we‘ll see those in the wild shortly!

**Reply by norwd** [reply]
I hope so 😄

**Reply by Schweinepriester** [reply]
actually!

the assets are already live! :D 😎 

e.g. https://github.githubassets.com/images/modules/profile/achievements/heart-on-your-sleeve-default.png:

![image](https://user-images.githubusercontent.com/2644614/200822987-d5ed0da8-ab45-483f-ac6c-70266c73bacd.png)

… now guessing the URL for the other one! :D

**Reply by Schweinepriester** [reply]
giving up for now, tried quite a lot:
based on the galaxy brain above it I should be "Open S" and then at most 4 other letters.
tried "src" or "srcs", stuff like that, haven't found it.

even hacked together this dirty script to try to guess it:
https://gist.github.com/Schweinepriester/e9fb757675eb2bdf1aa10d521e00c399

**Reply by Schweinepriester** [reply]
if you want to try:
https://github.githubassets.com/images/modules/profile/achievements/open-sREPLACEME-default.png

**Reply by norwd** [reply]
Open Sourcerer? :joy:

**Reply by norwd** [reply]
OMG IT IS OPEN SOURCERER!!!! https://github.githubassets.com/images/modules/profile/achievements/open-sourcerer-default.png

**Reply by Schweinepriester** [reply]
VERY NOICE, WORLD FIRST!

how have I not hit that with my many permutations oO

great job, I‘ll add it asap!

**Reply by Schweinepriester** [reply]
@norwd …unless you want to do the PR e.g. to change it to open sourcerer so you are listed as contributor in this repo, as attribution to you for finding it?

**Reply by norwd** [reply]
Added in #477

**Reply by Schweinepriester** [reply]
great stuff! 🎉 

now, if you find the third one (because so far they came in threes) without any clue whatsover, you win the internet :D

**Comment by parallels999** [comment]
https://www.youtube.com/shorts/1UgqjSCGslg

**Comment by MathieuSoysal** [comment]
ChatGPT respond that: 

![image](https://user-images.githubusercontent.com/43273304/218876504-d1c2e8fc-0540-4bb3-936a-2f9a793941cc.png)

Le badge "Heart On Your Sleeve" est un badge que les utilisateurs de GitHub peuvent ajouter à leur profil pour montrer leur soutien à des organisations à but non lucratif.

Pour ajouter le badge, l'utilisateur doit d'abord faire un don à une organisation à but non lucratif en utilisant GitHub's Sponsorships ou GitHub's Donations. Ensuite, l'utilisateur peut ajouter le badge "Heart On Your Sleeve" à son profil pour montrer son soutien à l'organisation.

Le badge "Heart On Your Sleeve" apparaîtra sur le profil de l'utilisateur avec le nom de l'organisation qu'il a soutenue. Cela peut aider à sensibiliser à l'organisation et encourager les autres à faire des dons également.

![image](https://user-images.githubusercontent.com/43273304/218876602-3037bd14-525e-4ab5-af6e-e40d10e7e12f.png)

Le badge "Open Sourcerer" est un badge de profil que les utilisateurs de GitHub peuvent obtenir lorsqu'ils contribuent à des projets open source. Ce badge est décerné aux utilisateurs qui ont au moins un dépôt public et ont effectué leur première contribution à un projet open source.

Pour obtenir le badge "Open Sourcerer", l'utilisateur doit:

Avoir un compte GitHub et un dépôt public.
Contribuer à un projet open source en soumettant une demande de tirage (pull request) ou en créant une issue.
Avoir la contribution acceptée par le propriétaire du projet open source.
Une fois que la contribution est acceptée, le badge "Open Sourcerer" sera automatiquement ajouté à leur profil GitHub. Le badge apparaîtra sur le profil de l'utilisateur avec le nombre de contributions qu'il a apportées à des projets open source, ce qui peut aider à montrer leur engagement envers la communauté open source.

**Reply by MathieuSoysal** [reply]
I don't know if is very useful ;x

**Reply by norwd** [reply]
For those who don't speak French:

> The "Heart On Your Sleeve" badge is a badge that GitHub users can add to their profile to show their support for nonprofit organizations.
>
> To add the badge, the user must first donate to a nonprofit using GitHub's Sponsorships or GitHub's Donations. Then the user can add the "Heart On Your Sleeve" badge to their profile to show their support for the organization.
>
> The "Heart On Your Sleeve" badge will appear on the user's profile along with the name of the organization they supported. This can help raise awareness of the organization and encourage others to donate as well.
>
> The "Open Sourcerer" badge is a profile badge that GitHub users can earn when contributing to open source projects. This badge is awarded to users who have at least one public repository and have made their first contribution to an open source project.
>
> To obtain the "Open Sourcerer" badge, the user must:
>
> Have a GitHub account and a public repository.
Contribute to an open source project by submitting a pull request or creating an issue.
Have the contribution accepted by the owner of the open source project.
Once the contribution is accepted, the "Open Sourcerer" badge will automatically be added to their GitHub profile. The badge will appear on the user's profile along with the number of contributions they have made to open source projects, which can help show their commitment to the open source community.

**Reply by norwd** [reply]
As @MathieuSoysal points out, a predictive text AI's answer isn't particularly useful, but if it's correct or at least close, these are at least simple steps to follow to test it.

I must say however that I think the prediction for "Open Sourcerer" would overlap with the "Pull Shark" achievement too much. I hope that when these new achievements come out they are a bit more interesting...

**Comment by Tynab** [comment]
To earn the Open Sourcerer achievement on GitHub, you need to make a qualifying contribution to an open source project on GitHub. Here are the steps you can follow:
1. Sign up for a GitHub account if you haven't already.
2. Find an open source project on GitHub that you would like to contribute to. You can search for open source projects using GitHub's Explore feature, or you can look for projects in specific programming languages or topics that interest you.
3. Fork the repository of the open source project that you want to contribute to. This creates a copy of the project in your own GitHub account that you can make changes to.
4. Make a qualifying contribution to the project. This can be anything from fixing a bug or adding a new feature, to improving documentation or submitting a new translation. Your contribution must be accepted and merged into the main repository by the project maintainers to count towards the Open Sourcerer achievement.
5. Once your contribution has been merged, you should receive a notification from GitHub that your contribution has been accepted. After 30 days of making your first contribution, you will earn the Open Sourcerer badge on your GitHub profile.
Note that GitHub has specific criteria for what constitutes a qualifying contribution, so be sure to review the contribution guidelines for the project you want to contribute to before getting started. Good luck!

**Reply by Schweinepriester** [reply]
https://github.com/Schweinepriester/github-profile-achievements/discussions/574#discussioncomment-5011674
